File: our-moments/src/main/java/com/gravity/ourmoments/entity/Notification.java
package com.gravity.ourmoments.entity;

import lombok.Data;
import java.time.LocalDateTime;

@Data
public class Notification {
    private Long notificationId;
    private Long userId;
    private String type; // COMMENT, FRIEND_REQUEST, NEW_POST
    private String content;
    private Long relatedId; // 关联ID（评论ID、好友关系ID、文章ID等）
    private Boolean isRead;
    private LocalDateTime createTime;
}

File: our-moments-android/app/src/main/java/com/gravity/ourmoments/data/model/Notification.kt
package com.gravity.ourmoments.data.model

import kotlinx.serialization.Serializable

@Serializable
data class Notification(
    val notificationId: Long,
    val content: String,
    val type: String,
    val relatedId: Long,
    val isRead: Boolean,
    val createTime: String
)